### ISSUE TITLE
Linking issue

### ISSUE THREAD

**Issue by Rinnegatamante** [psp2sdk/psp2sdk#66]
When i try to compile the vitahelloworld i get several linker errors (I'm using latest nightly build from Wololo.net):
```
cc2ccxC6.ltrans0.o:(.text+0xc0): undefined reference to `__aeabi_atexit'
cc2ccxC6.ltrans0.o:(.text+0x1d8): undefined reference to `exit'
main.o: In function `main':
main.c:(.text+0x254): undefined reference to `rand'
main.c:(.text+0x294): undefined reference to `rand'
main.c:(.text+0x2d4): undefined reference to `rand'
draw.o: In function `alloc_gpu_mem':
draw.c:(.text+0xac): undefined reference to `printf'
draw.c:(.text+0xd8): undefined reference to `printf'
draw.c:(.text+0xec): undefined reference to `printf'
draw.c:(.text+0x130): undefined reference to `printf'
draw.o: In function `init_video':
draw.c:(.text+0x1a8): undefined reference to `printf'
draw.o:draw.c:(.text+0x2a8): more undefined references to `printf' follow
draw.o: In function `clear_screen':
draw.c:(.text+0x4f0): undefined reference to `memset'
draw.o: In function `font_draw_stringf':
draw.c:(.text+0xa34): undefined reference to `vsnprintf'
collect2.exe: error: ld returned 1 exit status
make: *** [psp2helloworld.elf] Error 1
```

**Comment by 173210** [comment]
See the tutorial.
https://github.com/psp2sdk/psp2sdk/wiki/A.-How-to-Install-PSP2SDK#4-replace-prefix-in-psp2specsin-and-save-it-as-psp2specs-in-an-appropriate-directory

**Comment by 173210** [comment]
Identified the issue and fixed with psp2sdk/libs@2976635b02791290bcc4047b77506cfb9c4ff222.
